src/foam/u2/view/PopUpTitledView.js

```diff
@@ -0,0 +1,241 @@
+        width: 135px;
+        height: 40px;
+        border-radius: 2px;
+        background-color: #5e91cb;
+        cursor: pointer;
+        text-align: center;
+        color: #ffffff;
+        font-family: Roboto;
+        font-size: 14px;
+        line-height: 2.86;
+        letter-spacing: 0.2px;
+        margin-top: 5px;
+        margin-left: 293px;
+        margin-right: 20px;
+        margin-bottom: 20px;
+        float: left;
+      }
+
+      ^ .foam-u2-ActionView {
+        line-height: 0px;
+      }
+
+      ^ .foam-u2-ActionView:hover {
+        background-color: #5e91cb;
+        background: #5e91cb;
+      }
+
+      ^ .Button-Container {
+        margin: 0;
+        margin-top: 20px;
+        padding: 0;
+        overflow: hidden;
+      }
+
+      ^ .cancel-Button, .foam-u2-ActionView-back {
+        width: 135px;
+        height: 40px;
+        border-radius: 2px;
+        background-color: rgba(164, 179, 184, 0.1);
+        box-shadow: 0 0 1px 0 rgba(9, 54, 73, 0.8);
+        cursor: pointer;
+        text-align: center;
+        font-family: Roboto;
+        font-size: 14px;
+        line-height: 2.86;
+        letter-spacing: 0.2px;
+        margin-top: 5px;
+        margin-left: 20px;
+        margin-bottom: 20px;
+        position: fixed;
+        color: #093649;
+      }
+
+      ^ .foam-u2-ActionView-back:hover {
+        background-color: rgba(164, 179, 184, 0.1);
+        background: rgba(164, 179, 184, 0.1);
+      }
+
+      ^ .input-container {
+        margin-top: 20px;
+      }
+
+      ^ .pDefault {
+        margin-bottom: 8px;
+        margin-top: 0;
+        margin-left: 20px;
+        font-family: Roboto;
+        font-size: 14px;
+        font-weight: 300;
+        letter-spacing: 0.2px;
+        text-align: left;
+        color: #093649;
+      }
+
+      ^ .foam-u2-ActionView-closeButton {
+        width: 24px;
+        height: 24px;
+        margin: 0;
+        margin-top: -10px;
+        margin-right: 20px;
+        cursor: pointer;
+        display: inline-block;
+        float: right;
+        border: none;
+        background: transparent;
+        outline: none;
+      }
+
+      ^ .foam-u2-ActionView-closeButton:hover {
+        outline: none;
+        border: none;
+        background: transparent;
+      }
+
+      ^ .summary-heading {
+        padding: 10px 74px 0px 20px;
+        font-family: Roboto;
+        font-size: 14px;
+        font-weight: bold;
+        letter-spacing: 0.2px;
+        text-align: left;
+        color: #093649;
+      }
+    */}
+    })
+  ],
+
+  methods: [
+    function initE() {
+    this.SUPER();
+    var self = this;
+
+    this
+      .addClass(this.myClass())
+      .start()
+        .start().addClass('Message-Container')
+          .start().addClass('Change-Container')
+            .start().addClass('Change-Text').add(self.title).end()
+            .start()
+              .add(self.CLOSE_BUTTON)
+              .addClass('close-Button')
```

Comment:
Is adding a class here required? Aren't actions automatically assigned meaningful classes that you could use instead?